.readthedocs.yml

```diff
@@ -0,0 +1,14 @@
+version: 2
+
+mkdocs:
+  configuration: mkdocs.yml
+
+build:
+  os: ubuntu-20.04
+  tools:
+    python: "3.6"
+
+python:
+  install:
+    - requirements: pip_require.txt
+    - requirements: requirements.txt
```

Comment:
Need a new line at end of file.

Reply:
Good catch, thanks! Added in https://github.com/ibexa/documentation-user/pull/307/commits/ba80875891f191e6f425f5024b2fe18244e70e08